src/views/List/List.jsx

```diff
@@ -9,6 +9,12 @@ import Footer from '../../components/Footer/Footer';
 
 export default function List({ token }) {
   const [data, setData] = useState([]);
+  //this is just a duplicate of the data, so that the real data is never touched, its populated in the useEffect
```

Comment:
I love all the comments you've added, makes this super easy to read and understand